### DISCUSSION TITLE
AUV3 conversion to CLAP

### DISCUSSION THREAD

**Discussion by XtremeSounds** [free-audio/clap#98] [Ideas]
Anyone know if it would be easy to convert AUV3 to the CLAP format?   if so as this would open up ios apps to eb used on a windows PC.

**Comment by luciascarlet** [comment]
I can't really imagine this being possible on anything other than macOS on ARM unfortunately... The thing is, not only is the code (if you're dealing with an iOS app) highly architecture-specific, it also uses iOS-specific API's that only exist on iOS and have never been reverse-engineered for other operating systems in the sort of way Wine allows you to run Windows apps. The closest thing is Darling, a Darwin compatibility layer for Linux, which supports *very* basic macOS GUI apps but not much more, certainly not iOS which has its own set of API's (Cocoa Touch/UIKit as opposed to Cocoa/AppKit on macOS). Even if all of this existed, you'd still have to emulate the ARM architecture if you're running on an x86/64 PC, which you most likely are if you're using Windows.

**Comment by XtremeSounds** [comment]
Too bad I guess. How can developers be encouraged to make their ios apps available for windows 11?
